src/main/java/com/akademia/educators_back/controller/CommentController.java

```diff
@@ -0,0 +1,37 @@
+package com.akademia.educators_back.controller;
+
+import com.akademia.educators_back.service.impl.CommentServiceImpl;
+import com.akademia.educators_back.to.CommentTo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/comment")
+public class CommentController {
+    private CommentServiceImpl commentServiceImpl;
+
+    public CommentController(CommentServiceImpl commentService) {
+         this.commentServiceImpl = commentService;
+    }
+
+    @GetMapping("/api/commentList")
```

Comment:
Nasz endpoint wygląda teraz tak: "localhost8080/api/comment/api/commentList" czyli dziwnie. To działa tak, że to co mamy w @RequestMapping to jest główny zasób, a każdy kontroler osobno go rozwija dalej. Najlepiej przetestować to sobie Postmanem. 
W tym wypadku możemy dać samo "/list/" i wtedy docelowy endpoint będzie wyglądać "api/comment/list" czyli bez zbędnych powtórzeń